website/docs/guides/migration/versions/05-upgrading-to-v1.3.md

```diff
@@ -31,6 +40,8 @@ _GitHub discussion forthcoming_
 
 - **[Python models](building-models/python-models)** are natively supported in `dbt-core` for the first time, on data warehouses that support Python runtimes.
 
+- **[Metrics](building-a-dbt-project/metrics)** have been updated to reflect their new syntax for definition, as well as additional attributes that are now available.
+
 Docs in progress: [Issues labeled "dbt-core v1.3"](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.3%22+)
 
-- **[`node_color` now a supported `docs` attribute](/docs/reference/resource-configs/docs.md)**: You can add custom colors to the dbt docs lineage(DAG) graph. For the first time, you can physically visualize layers within the DAG, such as bronze, silver, and gold.
\ No newline at end of file
+- **[`node_color` now a supported `docs` attribute](/docs/reference/resource-configs/docs.md)**: You can add custom colors to the dbt docs lineage (DAG) graph. For the first time, you can physically visualize layers within the DAG, such as bronze, silver, and gold.
```

Comment:
so cool! removed the dbt docs lineage since dag was spelled out in the beginning of the doc.

```suggestion
- **[`node_color` now a supported `docs` attribute](/docs/reference/resource-configs/docs.md)**: You can add custom colors to the DAG. For the first time, you can physically visualize layers within the DAG, such as bronze, silver, and gold.
```